website/docs/patterns/form-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,236 @@
+
+- Long forms are more challenging for the user which can result in a lower completion rate.
+- If a field is optional, consider whether it should be included in the form at all. If the information isn't crucial to the experience or feature, why is it being collected at all?
+!!!
+
+#### Using dividers
+
+Dividers introduce more visual hierarchy and differentiation in long, complex forms.
+
+Only use dividers between sections of a form, not between fields or other smaller elements within a section. A divider should be used to break up different types of content and categories within a form.
+
+When using a divider, include a 24px gap above and below to separate the divider from surrounding form elements, otherwise it can appear "attached" to a specific section or field.
+
+![Spacing using dividers](/assets/patterns/form-patterns/divider-spacing.png =500x*)
+
+!!! Dont
+
+Dividers should not be used trailing the last section of a form between the fields and the button set or actions. Instead, the section spacing of 32px should be used.
+!!!
+
+## Width
+
+The width of a form and the fields it contains is largely dependent on the context the form is in, but adhering to these high-level guidelines can make a complex form approachable and introduce consistency in the UX.
+
+#### Form width
+
+A form should hug the content and the fields it contains.
+
+![Form width](/assets/patterns/form-patterns/form-width-example.png)
+
+#### Field width
+
+The width of a field within a form should be wide enough to account for the estimated width of the content it accepts. This gives the user an accurate sense of the length and type of the content the field accepts and is important in setting user expectations.
+
+#### Single column layout
+
+If a form uses a single-column layout, consider setting a maximum width on the form; this can generally be best achieved by using a size or unit that is **relative** the page, viewport, or container size. This can commonly be achieved by using a viewport width unit (`vw`), percentage width (e.g., `50%`), or a character unit (`ch`).
+
+!!! Info
+
+The web is a fluid medium; a relative width will adjust based on the viewport and container width. Designing and building a form with a recommended maximum and minimum width will help to conceptualize the graceful expansion and contraction of the viewport
+!!!
+
+<!-- Constrained vs unconstrained -->
+
+### Responsive properties
+
+As the viewport shrinks, the form width should expand relative to the viewport width, eventually occupying the entire width of the viewport or page. Horizontally grouped fields that result in a multi-column layout should stack vertically as the viewport shrinks.
+
+## Progressive disclosure
+
+Users want power, but also want simplicity. Progressive disclosure refers only displaying the most important information (or elements of a form) and conditionally showing additional fields depending on:
```

Comment:
> Users want power

This feels like an odd statement to me. What are you really trying to convey here?

Reply:
```suggestion
Users want power but also simplicity. Progressive disclosure refers to only displaying the most important information (or elements of a form) and conditionally showing additional fields depending on:
```

Reply:
I have some beefing up to do on this content captured in [this ticket](https://hashicorp.atlassian.net/browse/HDS-1610), but essentially what I'm trying to say is progressive disclosure is a method that can be used to different levels of complexity within an experience depending on a number of different factors (settings the user has enabled, past behavior, customizable settings, etc).